projects/fsastorefrontstyles/scss/components/cms/_enriched-responsive-banner.scss

```diff
@@ -3,8 +3,80 @@
     position: relative;
     width: 100%;
   }
+  .enriched-banner-text {
+    background-color: rgba(255, 255, 255, 0.7);
+    position: absolute;
+  }
+  .enriched-banner-text-strip {
+    display: flex;
+    flex-flow: column;
+    width: 100%;
+  }
+  .enriched-banner-title {
+    font-weight: var(--cx-font-weight-semi);
+  }
+  .box-title {
+    font-size: var(--cx-h1-font-size);
+    color: var(--cx-color-secondary);
+    @include media-breakpoint-down(md) {
+      font-size: 4vw;
+    }
+  }
+  .strip-title {
+    font-size: var(--cx-h3-font-size);
+    color: var(--cx-color-text);
+    padding-bottom: 1.2rem;
+    @include media-breakpoint-down(md) {
+      font-size: var(--cx-title-font-size);
+    }
+  }
+  .enriched-banner-details-box {
+    display: block;
+    margin-top: 1vw;
+    pointer-events: none;
+    font-size: var(--cx-h5-font-size);
+    @include media-breakpoint-down(sm) {
+      font-size: 3.5vw;
+    }
+  }
+  .enriched-banner-details-strip {
+    display: flex;
+    justify-content: center;
+  }
+  .enriched-banner-styled-text {
+    @extend .primary-button;
+    font-weight: var(--cx-font-weight-semi);
```

Comment:
Maybe use clamp for font-size so we don't have to use media queries.

Reply:
Here `font-size` is defined in the class `.primary-button`, so I don't see the point of using clamp to override that style..

Reply:
Font-wight is not needed here as there is no difference without it.